std/traits.d

```diff
@@ -44,6 +44,7 @@
  *           $(LREF hasElaborateDestructor)
  *           $(LREF hasIndirections)
  *           $(LREF hasMember)
+ *           $(LREF isStaticMember)
```

Comment:
This still needs to be changed, and your most recent commit has unrelated changes mixed in.